sim/battle.ts

```diff
@@ -3103,7 +3103,7 @@ export class Battle extends Dex.ModdedDex {
 		throw new UnimplementedError('canUltraBurst');
 	}
 
-	canZMove(pokemon: Pokemon): (AnyObject | null)[] | void {
+	canZMove(pokemon: Pokemon): ({move: string, target: string} | null)[] | void {
```

Comment:
please modify in `globals.ts` as well

Reply:
Is it necessary? Wouldn't it be enough to define an ``Interface`` in the same file?

Reply:
Or, I guess, ``import`` it to the ``globals.ts``?

Reply:
No, I mean just make sure the signatures of `canZMove` here and `globals.ts` match to avoid misleading developers. 

I'm going to work to eliminate `globals.ts`, but in the mean time I think its helpful if you can look at either here or `globals.ts` to figure out the signature, as opposed to having to look at both and merge in your head (which is what Typescript does)